creator-node/src/config.js

```diff
@@ -168,6 +168,55 @@ const config = convict({
     format: String,
     env: 'delegatePrivateKey',
     default: null
+  },
+
+  ethProviderUrl: {
+    doc: 'eth provider url',
+    format: String,
+    env: 'ethProviderUrl',
+    default: null
+  },
+  ethNetworkId: {
+    doc: 'eth network id',
+    format: String,
+    env: 'ethNetworkId',
+    default: null
+  },
+  ethTokenAddress: {
+    doc: 'eth token address',
+    format: String,
+    env: 'ethTokenAddress',
+    default: null
+  },
+  ethRegistryAddress: {
+    doc: 'eth registry address',
+    format: String,
+    env: 'ethRegistryAddress',
+    default: null
+  },
+  ethOwnerWallet: {
+    doc: 'eth owner wallet',
+    format: String,
+    env: 'ethOwnerWallet',
+    default: null
+  },
+  spOwnerWallet: {
+    doc: 'Service provider owner wallet',
+    format: String,
+    env: 'spOwnerWallet',
+    default: null
+  },
+  ethWallets: {
+    doc: 'all unlocked accounts from eth chain',
+    format: Array,
```

Comment:
@SidSethi Are these local only? Because if they aren't defined in a production env it will error

Reply:
would it still error with `default: null`?

Reply:
yeah it will. basically everything defined here also has to be defined in a .env file. if there's any values that are null it will error. there are two options. either we remove this from the config checking and use process.env.<variable> or we give it some dummy default value to get it to not complain

Reply:
ok fair. i made sure all have non-null defaults somewhere